flexget/plugins/input/myepisodes_list.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import re
+from itertools import chain
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.utils.soup import get_soup
+
+log = logging.getLogger('myepisodes')
+
+URL = 'http://www.myepisodes.com/'
+
+class MyEpisodesList(object):
+    """Creates an entry for each item in your myepisodes.com show list.
+
+    Syntax:
+
+    myepisodes_list:
+      username: <value>
+      password: <value>
+      strip_dates: <yes|no>
+      include_ignored: <yes|no>
+
+    Options username and password are required.
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'strip_dates': {'type': 'boolean', 'default': False},
+            'include_ignored': {'type': 'boolean', 'default': False},
+        },
+        'required': ['username', 'password'],
+        'additionalProperties': False,
+    }
+
+    @cached('myepisodes_list')
+    @plugin.internet(log)
+    def on_task_input(self, task, config):
+        if not task.requests.cookies:
+            username = config['username']
+            password = config['password']
+
+            log.debug("Logging in to %s ..." % URL)
+            params={
+                'username': username,
+                'password': password,
+                'action': 'Login'
+            }
+            loginsrc = task.requests.post(URL + 'login.php', data=params).content
```

Comment:
Need some exception handling here... What if we get a http 5xx, 3xx or timeout error?